### PR TITLE
Work branch

### PR DESCRIPTION
**Pull request by mikolajkus**
Whole project of credit-card is done. Give me feedback, what you think about it?

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,2 +1,86 @@
...
+}
+
+.credit-card-back .number.number {
+    width: 30%;
+    height: 100%;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0;
+    font-size: 20px;
+    font-weight: lighter;
```
Comment (mwanago) [inline style.css:84]: If you take a closer look, there is one CSS property missing that you should apply. This text should be in italic.

Screenshot from 2024-03-08 12-08-25

File: style.css [hunk 2]
```diff
@@ -1,2 +1,86 @@
...
+.credit-card-back .name {
+    width: 100%;
+    font-family: "Sacramento", cursive;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 20px;
+    margin: 15px 0 0 25px;
+}
+
+.credit-card-back .number.number {
```
Comment (mwanago) [inline style.css]: Use the trick with `.number.number` as a last resort. You could change this selector to
```
.credit-card-back .information-wrapper .number
```
and have the same specificity.

You will be styling a `.number` container inside the `.information-wrapper` that's inside `.credit-card-back` which seems to be exactly what you need here.

File: style.css [hunk 3]
```diff
@@ -1,2 +1,86 @@
...
+
+.information-wrapper > p:nth-child(1) {
+    font-weight: lighter;
+    text-transform: uppercase;
+}
+
+.information-wrapper > p:nth-child(2) {
+    margin-left: 10px;
+    font-size: 30px;
+}
```
Comment (mwanago) [inline style.css]: You could simplify `.information-wrapper > p:nth-child(x)` to just `.information-wrapper p:nth-child(x)` in this case.
Do you know why?

File: style.css [hunk 4]
```diff
@@ -1,2 +1,86 @@
+.credit-card  {
+    width: 500px;
+    height: 325px;
+    background-color: royalblue;
+    border-radius: 20px 20px 20px 20px;
+    display: flex;
+    flex-direction: column;
+    font-family: "Geologica", sans-serif;
+    font-weight: 300;
+    font-optical-sizing: auto;
```
Comment (mwanago) [inline style.css]: Why `font-optical-sizing`?

Comment (mikolajkus) [reply style.css]: oh i copied it and rest code from the google fonts. it isn't necessary of course. I will delete it.

File: style.css (hunk 3 above)
Comment (mikolajkus) [reply style.css]: Becasue the ```p``` elements are only childrens of ```.information-wrapper``` ?

File: style.css (hunk 2 above)
Comment (mikolajkus) [reply style.css]: You're right.

File: style.css (hunk 1 above)
Comment (mikolajkus) [reply style.css:84]: I didn't notice it.

File: style.css (hunk 3 above)
Comment (mwanago) [reply style.css]: @mikolajkus Yeah. By using `>` you specify that you want to style only the `p` elements that are only the direct children of the `.information-wrapper`. But all `p` elements inside `.information-wrapper` **are** the direct children, so no need for the `>` sign

**Review by mwanago — APPROVED** [review]

**Pull request merged**